specification/cognitiveservices/data-plane/Personalizer/preview/v1.1-preview.2/Personalizer.json

```diff
@@ -639,9 +639,50 @@
           }
         }
       },
+      "put": {
+        "tags": [
+          "ModelV1Dot1Preview2"
+        ],
+        "summary": "Put Model.",
+        "description": "Replace the existing model file for the Personalizer service.",
+        "operationId": "Model_Put",
+        "consumes": [
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "format": "binary",
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          },
+          "default": {
```

Comment:
Error response should contain a x-ms-error-code header. This is true for all of your error definitions, actually. 
Does the service support this header today? If not, it should and it should be added to the open api doc.